### ISSUE TITLE
Error when running passenger start natively on an Mac mini M1 version

### ISSUE THREAD

**Issue by joerichsen** [phusion/passenger#2323]
**Question 1: What is the problem?**

```gem install passenger``` works fine :-)

Running ```passenger start``` results in the following:

```
 [passenger_native_support.bundle] trying to compile for the current user (joe) and Ruby interpreter...
     (set PASSENGER_COMPILE_NATIVE_SUPPORT_BINARY=0 to disable)
     Compilation successful. The logs are here:
     /var/folders/0b/b90tgxhj55xcglg0np_9rkbm0000gn/T/passenger_native_support-wgu7ae.log
 [passenger_native_support.bundle] successfully loaded.
sh: line 1: 51381 Abort trap: 6           /Users/joe/.passenger/support-binaries/6.0.7/nginx-1.18.0 -c /var/folders/0b/b90tgxhj55xcglg0np_9rkbm0000gn/T/passenger-standalone.53jrep/nginx.conf -p /var/folders/0b/b90tgxhj55xcglg0np_9rkbm0000gn/T/passenger-standalone.53jrep -t 2>&1
*** ERROR: the Nginx configuration that Phusion Passenger Standalone generated internally contains problems. The error message returned by the Nginx engine is:

    dyld: Library not loaded: /usr/local/opt/pcre/lib/libpcre.1.dylib
      Referenced from: /Users/joe/.passenger/support-binaries/6.0.7/nginx-1.18.0
      Reason: no suitable image found.  Did find:
    	/usr/local/opt/pcre/lib/libpcre.1.dylib: mach-o, but wrong architecture
    	/opt/homebrew/Cellar/pcre/8.44/lib/libpcre.1.dylib: mach-o, but wrong architecture

```

**Question 2: Passenger version and integration mode**:

Open source passenger 6.0.7

**Question 3: OS or Linux distro, platform (including version)**:

macOS Big Sur 11.0.1 running on a Mac mini M1

**Question 4: Passenger installation method**:

[X] RubyGems, no Gemfile

**Question 5: Your app's programming language (including any version managers) and framework (including versions)**:
Ruby 2.3.8 (natively compiled to the M1 processor using asdf installed via home-brew)
Rails 3.2

**Question 6: Are you using a PaaS and/or containerization? If so which one?**
No

**Question 7: Anything else about your setup that we should know?**
No

**Comment by CamJN** [comment]
Can you check the actual arch of both: `/usr/local/opt/pcre/lib/libpcre.1.dylib` and `/opt/homebrew/Cellar/pcre/8.44/lib/libpcre.1.dylib`. They should be different, and one should match the nginx executable's arch.

**Comment by joerichsen** [comment]
Thank you for responding so quickly. 

Yesterday I installed ruby and passenger using the rosetta 2 version of homebrew. Today I reinstalled homebrew, ruby and passenger in the M1 native version.

I missed the fact that there was some files from the first homebrew install in my ```~/.passenger``` directory and that caused the problem mentioned.

Removing the ```~/.passenger``` directory and reinstalling passenger resolved the issue.
